sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/Choice.java

```diff
@@ -39,6 +43,35 @@ public final class Choice {
     @JsonProperty(value = "finish_reason", required = true)
     private CompletionsFinishReason finishReason;
 
+
+    static {
+        ChoicePropertiesHelper.setAccessor(new ChoicePropertiesHelper.ChoiceAccessor() {
+            @Override
+            public void setText(Choice choice, String text) {
+                choice.setText(text);
+            }
+
+            @Override
+            public void setIndex(Choice choice, int index) {
+                choice.setIndex(index);
+            }
+
+            @Override
+            public void setLogprobs(Choice choice, CompletionsLogProbabilityModel logprobs) {
+                choice.setLogprobs(logprobs);
+            }
+
+            @Override
+            public void setFinishReason(Choice choice, CompletionsFinishReason finishReason) {
+                choice.setFinishReason(finishReason);
+            }
+        });
+    }
```

Comment:
Where these helper methods generated or added by hand? If the latter, I think we should somehow figure out a way to separate generated code from custom code.